contrib/board_files/Makefile.am

```diff
@@ -1 +1,29 @@
-sysconf_DATA = @board@/libsoc_gpio.conf
+BOARD_FILES = $(shell find ./ -name \*.conf)
+
+install-data-local:
+	for p in $(BOARD_FILES); do \
+		echo $(MKDIR_P) "$(DESTDIR)$(pkgdatadir)/$$(dirname $$p)";\
```

Comment:
sure - i was trying to mimic the logic that automake generates, but as you can see it tedious.